.config/dotnet-tools.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "fake-cli": {
+      "version": "5.19.0",
```

Comment:
Decided to go with this approach due to error described in  https://github.com/fsharp/FAKE/issues/2447 .